control/controller_test.go

```diff
@@ -1,454 +0,0 @@
-package control
```

Comment:
So where is this test moving to?

Reply:
I thought that the entire controller + tests could go to `platform`